cadquery/occ_impl/geom.py

```diff
@@ -361,6 +361,19 @@ def __getitem__(self, rc: Tuple[int, int]) -> float:
         else:
             raise IndexError("Out of bounds access into 4x4 matrix: {!r}".format(rc))
 
+    def __repr__(self) -> str:
+        """
+        Generate a valid python expression representing this Matrix
+        """
+        list = [ [self.__getitem__((r,c)) for c in range(4)] for r in range(4)]
+        return "Matrix("+"".join(str(list)).replace("],","],\n       ")+")"
```

Comment:
1. Reusing built in `list` for your own variable name is not a good idea IMHO.
1. Your `list` is a list of lists of numbers. You apply `str(..)` to it. It becomes one string. Then you apply `"".join(..)` to this string. That does not change the string in any way.
1. Then you make string replacements. Why make replacements instead of formatting everything properly in the first place?

Maybe something like this instead (or whatever the Black formater transformes it into)?
```python
matrix_arg = \
  ",\n       ".join(
    (str([self.__getitem__((r,c)) for c in range(4)]) \
     for r in  range(4)))
return f"Matrix({matrix_arg})"   
```

Reply:
Thank you for the feedback. I agree with all of your comments and have made another pull request with the appropriate changes.
Cheers,
Roger